### DISCUSSION TITLE
Where is the admin site?

### DISCUSSION THREAD

**Discussion by singularity-s0** [OpenTreeHole/backend#15] [Q&A] [answered]
Looks like the site has been intentionally disabled. Why?

How are you supposed to test the development server without the admin site?

**Comment by hasbai** [comment, marked as answer]
Django 自带的管理后台难以定制，管理工作可以自行开发管理后台或直接在前端中进行。

开发调试工作可以使用默认的管理员帐号通过 Postman 或前端应用完成。
